### PR TITLE
Release 1.0.6 with trace span encoding fix

### PR DESCRIPTION
**Pull request by prashantv**
This is a cherry-pick off the dev branch, there are no other changes in this release.

### CODE REVIEW

**Comment by blampe** [comment]
🚢

File: tracing_internal_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+// THE SOFTWARE.
+
+package tchannel
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/uber/tchannel-go/typed"
```
Comment (akshayjshah) [inline tracing_internal_test.go]: nit: import groups...we can fix this later on dev.

**Comment by akshayjshah** [comment]
:shipit:

**Comment by abhinav** [comment]
LGTM

**Pull request merged**